apps/emqx/etc/vm.args.cloud

```diff
@@ -87,13 +88,27 @@
 ## Suggested stack size, in kilowords, for dirty IO scheduler threads.
 #+sssdio 40
 
-## Sets scheduler bind type.
-## Can be one of: u, ns, ts, ps, s, nnts, nnps, tnnps, db
-#+sbt db
-
 ## Sets a user-defined CPU topology.
 #+sct L0-3c0-3p0N0:L4-7c0-3p1N1
 
+## Sets scheduler bind type. Setting to 'db' may achieve lower message latency
+##  than the default 'u' in most scenarios.
+## If binding of schedulers is not supported by the OS, or no available CPU
+##  topology detected, the runtime system silently ignores the error, and start
+##  up using unbound schedulers 'u'.
++stbt db
+
+## Sets scheduler busy wait threshold. Setting to 'none' disables busy waiting.
+## This can significantly reduce the CPU usage observed by the OS, but may
+##  increase message latency in some scenarios.
++sbwt none
```

Comment:
I suggest don't touch busy wait for the defaults as usually we don't have much gain.

I also remember there is a OTP bug around it about dead locking scheduler.

Reply:
Some article shows the KVM or K8S uses the `CFS quota` to schedule threads, which may conflict Erlang's busy wait mechanism, but I am not good at the kernel stuff. See this link: https://elixirforum.com/t/performance-of-erlang-elixir-in-docker-kubernetes/21493